ui/pages/institutional/custody/custody.js

```diff
@@ -634,7 +632,7 @@ const CustodyPage = () => {
           </Box>
         </Box>
       )}
-    </>
+    </Box>
   );
 };
 
```

Comment:
Possible to test cover changes in this component ?

Reply:
Thanks for the feedback. I already did it with snapshots